src/main/webapp/data/userMapData.json

```diff
@@ -379,11 +388,14 @@
       "ISEN",
       "Lille 1 University",
       "Lille Catholic University",
+      "Paris 7",
```

Comment:
According to the list given in #8814 , shouldn't the name be "University of Paris 7" ?

Reply:
Hmm... let's use "Paris Diderot University (Paris 7)" instead :)